app/controllers/users_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    # @user = User.new(name: params[:name], email: params[:email], password: params[:password])
+    @user = User.new(user_params)
+
+    if @user.save
+      redirect_to user_path(@user)
```

Comment:
You ought to redirect to `new_user_path` as mentioned in the [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms) link which can be seen form this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_create.png)